### DISCUSSION TITLE
Popup dialog when receiving text message - does not work

### DISCUSSION THREAD

**Discussion by opi70** [BearWare/TeamTalk5#2402] [Q&A]
Hello, on Windows 11 "Popup dialog when receiving text message" does not work.
TeamTalk is running in background. Some other Windows are in foreground. When a text message reicives no popup chat window comes up. Is there a solution?  Greetings Thomas

**Comment by CoBC** [comment]
Hi,
did you check open windows with Alt-Tab? The pop-up dialog doesn't get focus anymore since version 5.12

**Reply by opi70** [reply]
Hi,
is in 5,12 or earlier the pop-up dialog working like my question?

**Comment by opi70** [comment]
Hi, yes, the chat window is there, but it is still in background. 
Usually I work in other programs and when a text messages comes, the chat window should popup in the foreground. Otherwise I can't see when a text message comes. Sound is not possible in our company.

**Comment by CoBC** [comment]
Yes, in version prior to 5.12, the pop-up works as you describe.
@bear101 Should we make an option to automatically focus this window in addition to current option to open it?
Maybe a combobox "when receiving a private message: open a dialogbox in background, open a dialogbox in forground, do nothing"

**Reply by opi70** [reply]
Ok, thank you!

**Comment by valiant8086** [comment]
I do see both sides of this actually. I think for the vast majority of cases, it's desired that it wouldn't steel focus, me included. I can be doing something that needs to not be interrupted, like playing a racing game or something like that and an incoming chat would throw me out and cause me to lose. If I wasn't gaming though I'd rather it steel focus. Too bad I can't teach it not to steel focus only in certain situations lol.

Anyway, so saying, methinks there is logic and value for people in having this be able to work either way.

**Reply by opi70** [reply]
Ah ok, I understand what you mean. Thank you!
